includes/polling/os/zmtel.inc.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+$temp_data = snmpwalk_cache_oid($device, 'deviceInfo', [], 'ZMTEL-ODU-MIB', 'zmtel');
```

Comment:
Can you not just do a multi get here for the 4 values you need rather than working additional data?

Reply:
fixed